typings/index.d.ts

```diff
@@ -1049,7 +1049,7 @@ declare module 'discord.js' {
   }
 
   // tslint:disable-next-line:no-empty-interface - Merge RateLimitData into RateLimitError to not have to type it again
-  interface RateLimitError extends RateLimitData { }
+  interface RateLimitError extends RateLimitData {}
```

Comment:
Watch out for this change (and others caused by vscode's default formatter) but why don't you address the comment while you're at it?

Reply:
The linter fixes all the formating during commit. That comment isn't mine.

Reply:
I know the comment isn’t yours but you could address it I guess
But those formatting issues are caused by vscode’s default formatter, just use prettier

Reply:
Seems like they were fixed in a later commit, nvm

Reply:
The current formating is the correct one as it is same in the master. You commented on it just after rebase when this change was introduced. Since, no commit happened during rebase the linter couldn't have fixed it. It fixed it in the later commits, that's why this suggestion is showing as outdated.

I don't know what you want me to address about the comment. Could you be a little more specific?

Reply:
Ah alright my bad then
Since the comment says “Merge RateLimitData into RateLimitError to not have to type it again” you could address that since you’re touching interfaces and typings, it’d be in line with the theme of the PR imo